recipes/pypalettes/recipe.yaml

```diff
@@ -0,0 +1,58 @@
+context:
+  name: pypalettes
+  version: "0.1.4"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name | replace("-", "_") }}/${{ name | replace("-", "_") }}-${{ version }}.tar.gz
```

Comment:
It seems like all of the `replace` logic can be removed

Reply:
My bad, old habit, thanks for pointing out!